pkg/client/dubbo/dubbo_client.go

```diff
@@ -167,7 +159,7 @@ func (dc *DubboClient) check(key string) bool {
 	}
 }
 
-func (dc *DubboClient) create(key string, dm *DubboMetadata) *dg.GenericService {
+func (dc *DubboClient) create(key string, dm config.IntegrationRequest) *dg.GenericService {
```

Comment:
renam dm to another, dm is mean DubboMetadata.

Reply:
ok.